tests/test-javalin-jsonb/src/main/java/org/example/myapp/web/GetBeanForm.java

```diff
@@ -3,14 +3,15 @@
 import java.util.List;
 import java.util.Set;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
This looks like the incorrect import?  Should it be io.avaje.validation.constraints.NotNull ?